src/common/translations/localize.ts

```diff
@@ -79,37 +94,19 @@ export const computeLocalize = async (
       cache._localizationCache[messageKey] = translatedMessage;
     }
 
-    const argObject = {};
-    for (let i = 0; i < args.length; i += 2) {
-      argObject[args[i]] = args[i + 1];
+    let argObject = {};
+    if (args[0] && typeof args[0] === "object") {
```

Comment:
Also check if args.length == 1 ? 
```suggestion
    if (args.length === 1 && typeof args[0] === "object") {
```